rasa/telemetry.py

```diff
@@ -574,8 +602,8 @@ def strip_sensitive_data_from_sentry_event(
         _unused_hint: some hinting information sent alongside of the event
 
     Returns:
-        the event without any sensitive / PII data or `None` if the event should
-        be discarded.
+        the event without any sensitive / PII data or `None` if the event constitutes
+        an `ImportError` which should be discarded.
```

Comment:
This amendment in the docstring was added in the wrong function - you should move it to `filter_errors`.